src/main/java/cinema/service/impl/MovieServiceImpl.java

```diff
@@ -10,7 +10,7 @@
 @Service
 public class MovieServiceImpl implements MovieService {
     @Inject
-    private static MovieDao movieDao;
+    MovieDao movieDao;
```

Comment:
```suggestion
    private MovieDao movieDao;
```
я мала на увазі видалити static а не всі модифікатори доступу)